### ISSUE TITLE
Netflix background service dont start after 10 minutes

### ISSUE THREAD

**Issue by MisterT87** [CastagnaIT/plugin.video.netflix#451]
#### Your Environment
- Netflix add-on version: 0.16.3
- Operating system version/name: Manjaro deepin Linux x64 (Arch based) with Kodi 18.5
- Device model:  Intel N4200 (quad core), 4 GB RAM, Intel HD Graphics 505 (Broxton)

Used Operating system:
* [ ] Android
* [ ] iOS
* [ X ] Linux
* [ ] OSX
* [ ] Raspberry-Pi
* [ ] Windows

### Describe the bug
Background services dont start and dont get any errors (as far as I can say). So I cant test anything within kodi.
Tried to wait 10 for background services but w/o success.

#### Expected behavior
Get a notification about netflix background services started

#### Actual behavior
No background services available and also dont get any notifications or errors within kodi (wait for 10 minutes)

### Debug log
The debug log can be found from this link:
[https://pastebin.com/DpT9dZYh](https://pastebin.com/DpT9dZYh)

#### Installation
* [ X ] I'm using other Netflix Repo 
* [ ] I'm using a different source 

#### Screenshots

**Comment by CastagnaIT** [comment]
"I'm using other Netflix Repo" is checked, what repository?
Did you follow the installation instructions?
Did you install pycryptodomex?

**Comment by MisterT87** [comment]
> "I'm using other Netflix Repo" is checked, what repository?

Sorry misread - using "repository.castagnait-1.0.0.zip"

> Did you follow the installation instructions?

followed the raspbian instructions but used pacman instead of apt-get. 
And also tried to install pycryptodomex with sudo and w/o sudo (pip install --user pycryptodomex) - but both w/o success.

> Did you install pycryptodomex?

Yes (cant paste out from shell -> its another device)

**Comment by CastagnaIT** [comment]
seems almost like a problem with your python on linux but i don't know
it looks like netflix service not being called up for startup
not having any printed errors, i don't know how to help you..

all the other addons work?
have you tried reinstalling kodi yet?

**Comment by MisterT87** [comment]
I forgot to say: First time when I installed 0.16.3 in kodi then background services came up.
after a kodi restart i had this problem.

> all the other addons work?

on my kodi.log above maybe you see some addon errors - Thats because I reinstalled my Linux / Kodi and imported whole .kodi dir (where all addon data etc is in) from another machine and on that time I created that log file not all addons was up2date and I had to renew some old API stuff...

TL DR: for now all other kodi addons work just fine, maybe one:
official youtube addon from old bromix author - in general this addon works - but really slow load times when navigating to other pages (about 6-7 minutes per change)

> have you tried reinstalling kodi yet?

See above - TL DR: This is a fresh linux / kodi installed but with imported .kodi (userdata etc) dir from another machine

**Comment by networxnet** [comment]
Same issue for me.

System:
QNAP TS-453Be based an Ubuntu 18.04.3. LTS (Linux 4.14.24-qnap)
Celeron J3455, 4 cores, 10 GB, HD Graphics 500


> what repository?
repository.castagnait-1.0.0.zip
> Did you follow the installation instructions?
yes
> Did you install pycryptodomex?
yes, pycryptdome 3.4.3 installed.

Kodi installation is fresh, other addon work also when using py dependencies like myconnpy or cryptopy.

**Comment by MisterT87** [comment]
I got 2 new big hints from a friend of mine (python guru) - but i cant test today coz of time (I'll do it tommorow)
and then I will explain it detailled and if its fixed or not

but one of this has to do with ipv6 other one is more complex - so @networxnet if u also using ipv6 on your machine stay tuned :D

**Comment by CastagnaIT** [comment]
@MisterT87 let me know if your solution works

in case of negative results, do this:
you said that on the first install the service is started, can you do an experiment?

-Rename the hidden kodi data folder, with another name (so after you can restore your data)
-Start Kodi (kodi start from scratch)
-Enable Kodi debug
-Install this [plugin.video.netflix_0.16.3_20200125.zip](https://github.com/CastagnaIT/plugin.video.netflix/files/4112427/plugin.video.netflix_0.16.3_20200125.zip) this is a modified version that have debug in verbose already enabled
-After service start, copy the log and rename it as first_log
-Reboot Kodi, try wait if service starts and save the second log

let's try to see if we can see anything more...

**Comment by MisterT87** [comment]
@CastagnaIT Sure I wll...
But I need 1 more info, is ipv6 implemented in your netflix addon - or do it just ignore ipv6 and only using ipv4 ?
with this I can backtrace it faster

But really I hate python lib shit :D (especially on linux - dont know if windows also had such problems) but sometimes you have use python2 and python3 same machine (really messed up)

**Comment by CastagnaIT** [comment]
this addon doesn't use such low level communication,
for http communication relies on "requests" module: https://github.com/psf/requests
should support both ip versions

**Comment by MisterT87** [comment]
@CastagnaIT yes request addon supports definitely ipv4 AND ipv6 !

TL DR: Didnt change anything and problem is solved.

Just booted my machine and started kodi only one addon update popped up: TwitchTV (Source: [Github](https://github.com/MrSprigster/Twitch-on-Kodi) 
and then Netflix background services came up (3 seconds after kodi start) - Video Playback also working fine now.
Tested it 3 times...

Dont know exactly why problems are gone - my thinkings: previous TwitchTV version has a bad bug and fucked up my python libs
Anyway working now...

@networxnet which kodi version are you using and kodi version ?
and please upload your kodi.log file (See: [https://kodi.wiki/view/Log_file/Easy](https://kodi.wiki/view/Log_file/Easy ))

**Comment by CastagnaIT** [comment]
I'm glad you solved
was a strange behaviour...

**Comment by networxnet** [comment]
Problem still exists for me. Reboot was without sucess and i'm not using twitchTV. Try to get the debug information in the next days.
